pkg/sql/model.go

```diff
@@ -19,22 +19,69 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path"
 	"path/filepath"
+	"strings"
 
 	pb "sqlflow.org/sqlflow/pkg/proto"
 	"sqlflow.org/sqlflow/pkg/sqlfs"
 )
 
+const modelZooDB = "sqlflow"
+const modelZooTable = "sqlflow.trained_models"
+
 type model struct {
 	workDir     string // We don't expose and gob workDir; instead we tar it.
 	TrainSelect string
 }
 
-// save creates a sqlfs table if it doesn't yet exist, and writes the
+func (m *model) save(modelURI string, session *pb.Session) error {
+	if strings.Contains(modelURI, "://") {
+		uriParts := strings.Split(modelURI, "://")
+		if len(uriParts) == 2 {
+			// oss:// or file://
+			if uriParts[0] == "file" {
+				dir, file := path.Split(uriParts[1])
+				return m.saveTar(dir, file)
+			} else if uriParts[0] == "oss" {
+				return fmt.Errorf("save model to oss is not supported now")
+			}
+		} else {
+			return fmt.Errorf("error modelURI format: %s", modelURI)
+		}
+	}
+	db, err := NewDB(session.DbConnStr)
```

Comment:
Is it possible to unify `sqlfs` with `modelURI`? For example, can we make `sqlfs://database_name/table_name` behave the same as `table_name`? This seems to ensure consistency.

Reply:
`oss://` and `file://` are well know URI prefixes. We do not intend to invent a new one.